cnxarchive/views/exports.py

```diff
@@ -77,9 +77,10 @@ def get_export(request):
                                " filename*=UTF-8''{fname}".format(
                                        fname=encoded_filename)
     resp.body = file_content
+    slug_title = '-'.join(encoded_filename.split('-')[:-1])
```

Comment:
Would be good to add a comment saying that this is removing the version from the slug.